### DISCUSSION TITLE
Measurands slow to update

### DISCUSSION THREAD

**Discussion by carlosandtheo** [lbbrhzn/ocpp#1093] [Q&A]
Seeking suggestions how to increase the rate that measurands are updated in the OCPP integration, mine seem to only update every 15min's

**Comment by madebylk** [comment]
I am having the same issue. Did you find any solutions for this problem? Should it be raised as an issue instead of Q&A?

**Reply by Animalector** [reply]
mine is updating every 5 mins, but as a live dashboard view, that's not frequent enough. I looked at the files and couldn't see a 5 min update timer anywhere...

**Reply by Animalector** [reply]
actually, I had the vendor update the firmware on the EVSwitch Supply Equipment, and it's updating roughly every minute now. It must be a configuration of the EVSE...
